### PR TITLE
AssetPanda&One.UF theme fix

### PR DESCRIPTION
**Pull request by MohammadEImmam**
AssetPanda tables were not showing as dark, resulting in a very messed up theme.

### CODE REVIEW

**Comment by Gusted** [comment]
Please post screenshots.

**Comment by MohammadEImmam** [comment]
oneUF B4
ONEUF After
AP After
AP B4


Attached are before and after screenshots of the websites.

**Comment by MohammadEImmam** [comment]
> Please post screenshots.

Posted ^^

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -11411,6 +11411,18 @@ div.art-Header-jpeg
...
 
+login.assetpanda.com
+
+CSS
+tr {
+    background-color:#1c1c1b;                                                                        
+}
+.main-container {
+    background-color:#1c1c1b !important;                                                                        
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11422]:
```suggestion
tr,
.main-container {
    background-color: var(--darkreader-neutral-background) !important;                                                                     
}
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**